src/components/ItaliaTheme/Blocks/ArgumentsInEvidence/Block.jsx

```diff
@@ -75,6 +75,7 @@ const Block = ({
         )}
         {argument && (
           <CardReadMore
+            className="darker"
```

Comment:
Ho provato ad usare classi bootstrap ma l'html viene generato da un componente esterno di react italia e viene sovrascritto il comportamento all'interno

Reply:
Ma aspetta, il problema che ha segnalato baio sull'accessibilità su che sito lo ha visto? perchè sull'agid base va bene, quelli nel ticket mi sembrano colori di un altro sito. 
In quel caso, tutti i link di quel sito non superano il controllo sull'accessibilità, quindi probabilmente c'è (solo in quel sito) da modificare il colore dei link attraverso la variabile $link-color. Di default è $link-color: $primary, ma in quel sito si può provare a fare $link-color: darken($primary,30), aggiustando magari il valore di incremento.